lib/diagnostic.rb

```diff
@@ -43,6 +43,10 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    CSV.foreach('filename.csv',
+                headers: true,
+                header_converters: -> (h) { h.downcase.to_sym }) do |filename_row|
+          #{|filename_row| filename_row + " "}
+          #{|row_a, row_b| row_a + row_b}
```

Comment:
Almost. This wouldn't be a CSV though. It would be a File.open

```ruby
File.open(filename) do |file|
   file.each do |line|
      line.chomp + ' ' 
   end
end
```